src/Tools/dotnet-watch/BrowserRefresh/src/WebSocketScriptInjection.cs

```diff
@@ -13,22 +13,18 @@ namespace Microsoft.AspNetCore.Watch.BrowserRefresh
     /// Helper class that handles the HTML injection into
     /// a string or byte array.
     /// </summary>
-    public class WebSocketScriptInjection
+    public static class WebSocketScriptInjection
     {
         private const string BodyMarker = "</body>";
+        internal const string WebSocketScriptUrl = "/_aspnetcore-dev-tools/browser-refresh.js";
```

Comment:
Do you think we should consider standardizing on `/_framework/*` for system-generated URLs? I know that means we have to make sure we don't clash within that namespace, but it means there's only one top-level space that developers have to account for if they are trying to make sure framework-generated stuff is allowed through whatever other router customizations they are doing.

Reply:
That's a great idea and a good way to to ensure user and framework endpoints don't stomp on each other. I've updated my PR to include this feedback.